hack/gen_tilt_settings.sh

```diff
@@ -25,7 +25,7 @@ function get_latest_release() {
     release="$(curl -H "Authorization: token ${GITHUB_TOKEN}" -sL "${1}")" || ( set -x && exit 1 )
   fi
   # This gets the latest release as vx.y.z , ignoring any version with a suffix starting with - , for example -rc0
-  release_tag="$(echo "$release" | jq -r "[.[].tag_name | select( startswith(\"${2:-""}\")) | select(contains(\"-\")==false)] | max ")"
+  release_tag="$(echo "$release" | jq -r "[.[].tag_name | select( startswith(\"${2:-""}\"))] | max ")"
```

Comment:
nit: please fix the comment above so it follows the code

Reply:
ah, yes, it is unfortunate that I run e2e test which is 3h+ long...

Reply:
I will wait for it to pass, and re-run only integration test here after I fix it.

Reply:
@lentzi90 fixed it, and re-run e2e again (the previous run failed due to an issue/flake unrelated to these changes)

Reply:
Thanks!
(For some reason I cannot mark this thread as resolved...)

Reply:
Resolved now